dependencies/pom.xml

```diff
@@ -107,6 +135,93 @@
         <artifactId>jsr305</artifactId>
         <version>${jsr305.version}</version>
       </dependency>
+
+      <dependency>
+        <groupId>com.google.gwt</groupId>
+        <artifactId>gwt-dev</artifactId>
```

Comment:
gwt-dev is really bad-designed and has a lot of internal conflicts among library versions. So, we have to override them explicitly. I understood that we can get rid of it - but again, think that is out of scope for now.